master/getting-started/kubernetes/tutorials/app-layer-policy/index.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Application layer policy tutorial
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/tutorials/app-layer-policy/'
+---
+
+The included demo sets up a microservices application, then demonstrates the use of application
+layer policy to mitigate some common threats.
+
+## Pre-requisites
+
+To create a Kubernetes cluster which supports application layer policy, follow
+one of our [getting started guides]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes),
+and [enable application layer policy]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/app-layer-policy).
+
+You will also need the [calicoctl]({{site.baseurl}}/{{page.version}}/usage/calicoctl/install) command line tool.
+
+### Install the demo application
+
+We will use a simple microservice application to demonstrate Calico application layer policy.  The 
+[YAO Bank] application creates a customer-facing web application, a microservice that serves up
+account summaries, and an [etcd] database.
+
+```bash
+kubectl apply -f \
+{{site.url}}/{{page.version}}/getting-started/kubernetes/tutorials/app-layer-policy/manifests/10-yaobank.yaml
+```
+
+> **Note**: You can also
+> [view the manifest in your browser](manifests/10-yaobank.yaml){:target="_blank"}.
+{: .alert .alert-info}
+
+When the demo application has come up, you will see 3 pods.
+
+    kubectl get pods
+    
+    NAME                        READY     STATUS    RESTARTS   AGE
+    customer-2809159614-qqfnx   3/3       Running   0          21h
+    database-1601951801-m4w70   3/3       Running   0          21h
+    summary-2817688950-g1b3n    3/3       Running   0          21h
+    
+There is a Kubernetes ServiceAccount for each microservice in the application (in addition to the `default` account).
+
+    kubectl get serviceaccount 
+
+    NAME       SECRETS   AGE
+    customer   1         21h
+    database   1         21h
+    default    1         21h
+    summary    1         21h
+
+Notice also that Istio CA will have created a secret of type `istio.io/key-and-cert` for each 
+service account.  These keys and X.509 certificates are used to cryptographically authenticate 
+traffic in the Istio service mesh, and the corresponding service account identities are used by
+Dikastes in authorization policy.
```

Comment:
s/Dikastes/Application Layer Policy/

Reply:
I went for "Calico."

Application Layer Policy isn't really and entity that can take actions